thali/thalireplicationmanager.js

```diff
@@ -157,18 +166,19 @@ function syncRetry(peer) {
   }
 
   this._emitter.disconnect(peer.peerIdentifier, function (err) {
-    if (err) {
-      this.emit(ThaliReplicationManager.events.DISCONNECT_ERROR, err);
-    } else {
-      syncPeer.call(this, peer);
-    }
+    console.log('Disconnect error with error: %s', err);
```

Comment:
What if we didn't get an error on the disconnect? In that case won't err be null? Shouldn't we check for that and only run line 169 and 170 if there is an actual error?